File: src/test/fixtures/kotlin/ParameterCount.kt
// clean — not flagged by parameter count
fun cleanFewParams(a: Int, b: Int): Int {
    return a + b
}

// flagged — parameter count (medium)
fun flaggedManyParams(a: Int, b: Int, c: Int, d: Int, e: Int, f: Int): Int {
    return a + b + c + d + e + f
}

// flagged — parameter count (high)
fun flaggedTooManyParams(a: Int, b: Int, c: Int, d: Int, e: Int, f: Int, g: Int, h: Int, i: Int): Int {
    return a + b + c + d + e + f + g + h + i
}

File: src/test/fixtures/cpp/parameter_count.cpp
// clean — not flagged by parameter count
int cleanFewParams(int a, int b) {
    return a + b;
}

// flagged — parameter count (medium)
int flaggedManyParams(int a, int b, int c, int d, int e, int f) {
    return a + b + c + d + e + f;
}

// flagged — parameter count (high)
int flaggedTooManyParams(int a, int b, int c, int d, int e, int f, int g, int h, int i) {
    return a + b + c + d + e + f + g + h + i;
}
